client/src/components/Button/Button.js

```diff
@@ -1,55 +1,22 @@
 import React from "react";
 import PropTypes from "prop-types";
+import styled from "styled-components/macro";
 
-/**
- * Primary UI component for user interaction
- */
-const Button = ({ primary, backgroundColor, size, label, ...props }) => {
-  const mode = primary
-    ? "storybook-button--primary"
-    : "storybook-button--secondary";
-  return (
-    <button
-      type="button"
-      className={["storybook-button", `storybook-button--${size}`, mode].join(
-        " "
-      )}
-      style={backgroundColor && { backgroundColor }}
-      {...props}
-    >
-      {label}
-    </button>
-  );
+const Button = styled.button`
+  height: 40px;
+  width: 330px;
+  border-radius: 15px;
+  background: var(--primary-color);
+  border: 2px solid var(--secondary-color);
+  cursor: pointer;
+  color: var(--secondary-color);
+  font-size: 29px;
+  font-family: "Heiti SC", sans-serif;
```

Comment:
you don't need this here once you set the font-family in the globalStyle

Reply:
only font-family is set in globalStyle.
Cursor, color and font-size are only set in Button.js. Is there something I don´t see?

Reply:
i just meant font-family :)